src/js/mixins/__tests__/SaveStateMixin-test.js

```diff
@@ -44,6 +44,16 @@ describe('SaveStateMixin', function () {
 
   describe('#componentWillUnmount', function () {
 
+    it('should call #saveState', function () {
+      this.instance.saveState = jasmine.createSpy();
+      this.instance.componentWillUnmount();
+      expect(this.instance.saveState).toHaveBeenCalled();
+    });
+
+  });
+
+  describe('#saveState', function () {
```

Comment:
for all the tests inside of this `describe`, can we change `componentWillUnmount` to `saveState`? this will actually test `saveState` instead of `componentWillUnmount`